Doc/library/textwrap.rst

```diff
@@ -17,31 +17,63 @@ If you're just wrapping or filling one or two text strings, the convenience
 functions should be good enough; otherwise, you should use an instance of
 :class:`TextWrapper` for efficiency.
 
-.. function:: wrap(text, width=70, **kwargs)
+.. function:: wrap(text, width=70, initial_indent="", subsequent_indent="", \
+                   expand_tabs=True, replace_whitespace=True, \
+                   fix_sentence_endings=False, \
+                   break_long_words=True, drop_whitespace=True, \
+                   break_on_hyphens=True, tabsize=8, \
+                   *, max_lines=None, **kwargs)
```

Comment:
Where is *placeholder*? What does *kwargs* include?

Reply:
I thought that *placeholder* only has an effect in :func:`shorten`?

`**kwargs` is there just because the actual function signature is like this:

```python
def wrap(text, width=70, **kwargs): ...
```

Any keyword argument will be accepted, but only correctly spelled kwargs will have an effect. I added an additional note to more specifically highlight this behavior; it adds on to the paragraphs where these kwargs are listed:

> Be careful not to misspell keyword arguments. Misspelled keyword arguments will silently have no effect, and an 
> exception will not be raised.

Reply:
I don't think that's the case, the kwargs are passed to TextWrapper, which will not accept unknown keyword args:

```
/usr/local/Cellar/python@3.9/3.9.1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/textwrap.py in shorten(text, width, **kwargs)
    403         'Hello [...]'
    404     """
--> 405     w = TextWrapper(width=width, max_lines=1, **kwargs)
    406     return w.fill(' '.join(text.strip().split()))
    407

TypeError: __init__() got an unexpected keyword argument 'zz'
```

Reply:
`placeholder` can apply to all 3 functions because it depends on `max_lines` -- if max_lines is specified and results in removing of some lines, placeholder will be added per docs.